src/chttpd_db.erl

```diff
@@ -248,9 +264,21 @@ delete_db_req(#httpd{}=Req, DbName) ->
         throw(Error)
     end.
 
+get_db_options(DbName) ->
+    IsReplicatorDb = DbName == config:get("replicator", "db", "_replicator"),
+    IsUsersDb = DbName ==config:get("chttpd_auth", "authentication_db", "_users") orelse
+    binary_to_list(mem3:dbname(DbName)) == config:get("chttpd_auth", "authentication_db", "_users"),
```

Comment:
@kxepal I was looking at tidying this up and realised I made two mistakes here. Firstly, there is no need for the `binary_to_list(mem3:dbname(DbName))` comparison because that would only be necessary if DbName was for an individual shard - that will never be the case here so that check can be removed.

Secondly, there is no need to check for the replicator DB here because chttpd does nothing with that information.

I'll add some commits to fix those things up.